lib/matplotlib/figure.py

```diff
@@ -2398,15 +2402,29 @@ def set_layout_engine(self, layout=None, **kwargs):
 
         Parameters
         ----------
-        layout: {'constrained', 'compressed', 'tight'} or `~.LayoutEngine`
-            'constrained' will use `~.ConstrainedLayoutEngine`,
-            'compressed' will also use ConstrainedLayoutEngine, but with a
-            correction that attempts to make a good layout for fixed-aspect
-            ratio Axes. 'tight' uses `~.TightLayoutEngine`.  Users and
-            libraries can define their own layout engines as well.
+        layout: {'constrained', 'compressed', 'tight', 'none'} or \
+`LayoutEngine` or None
+
+            - 'constrained' will use `~.ConstrainedLayoutEngine`
+            - 'compressed' will also use `~.ConstrainedLayoutEngine`, but with
+              a correction that attempts to make a good layout for fixed-aspect
+              ratio Axes.
+            - 'tight' uses `~.TightLayoutEngine`
+            - 'none' removes layout engine.
+
+            If `None`, the behavior is controlled by :rc:`figure.autolayout`
+            (which if `True` behaves as if 'tight' were passed) and
+            :rc:`figure.constrained_layout.use` (which if true behaves as if
+            'constrained' were passed).  If both are true,
```

Comment:
Consistent with above:
```suggestion
            :rc:`figure.constrained_layout.use` (which if `True` behaves as if
            'constrained' were passed).  If both are `True`,
```